src/main/resources/api/data-repository-openapi.yaml

```diff
@@ -3017,14 +3017,12 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/ErrorModel'
-  /api/repository/v1/datasets/{id}/snapshotRequests:
+  /api/repository/v1/snapshotRequests:
     get:
       tags:
-        - datasets
+        - SnapshotBuilderRequest
```

Comment:
I was expecting this term to be `SnapshotRequest` not `SnapshotBuilderRequest`. In my mind, the user isn't making a request _to_ the snapshot builder, they're making a request to create a snapshot.